### ISSUE TITLE
react-native-fading-slides is giving error React.createClass is not a function.

### ISSUE THREAD

**Issue by satishpalli** [chagasaway/react-native-fading-slides#2]
react-native-fading-slides is giving error React.createClass is not a function.( i am using latest react-native version 0.28). Please let me know how to resolve it

**Comment by chagasaway** [comment]
Hey @satishpalli, thanks for open this issue.

Seems like React-Native changed a little bit from the last release that I have made for this package (8 months ago), and now **React** should be require from _react_ package and not from _react-native_ as was before.

I just published a new version for _react-native-fading-slides_ on npm (**0.1.4**) that fixes this issue.

Let me know if that solved for you (there's as well an example project using react-native 0.28 here in the repo, that I have used the package without any issues).

Best regards,
